src/ucp/rndv/rndv_put.c

```diff
@@ -573,6 +573,21 @@ static void ucp_proto_rndv_put_mtype_frag_completion(uct_completion_t *uct_comp)
     ucp_proto_rndv_ppln_send_frag_complete(req, 1);
 }
 
+static ucs_memory_type_t
```

Comment:
Does it make sense to add a simple unit test into `test_ucp_am_nbx_rndv_ppln`?

Reply:
Added two test cases.